lib/Sema/TypeCheckConstraints.cpp

```diff
@@ -1413,20 +1435,21 @@ void ConstraintSystem::print(raw_ostream &out) const {
   // Print all type variables as $T0 instead of _ here.
   PrintOptions PO;
   PO.PrintTypesForDebugging = true;
-  
-  out << "Score: " << CurrentScore << "\n";
+
+  out << "Score:";
+  CurrentScore.print(out);
 
   for (const auto &contextualTypeEntry : contextualTypes) {
-    auto info = contextualTypeEntry.second.first;
-    out << "Contextual Type: " << info.getType().getString(PO);
-    if (TypeRepr *TR = info.typeLoc.getTypeRepr()) {
-      out << " at ";
-      TR->getSourceRange().print(out, getASTContext().SourceMgr, /*text*/false);
+    if (auto info = contextualTypeEntry.second.first; !info.getType().isNull()) {
```

Comment:
@amritpan Windows doesn't like this syntax - `C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift\lib\Sema\TypeCheckConstraints.cpp(1442): error C2429: language feature 'init-statements in if/switch' requires compiler flag '/std:c++17'`